server/src/signin-button.js

```diff
@@ -0,0 +1,38 @@
+const React = require("react");
+const PropTypes = require("prop-types");
+const { Localized } = require("fluent-react/compat");
+const sendEvent = require("./browser-send-event.js");
+
+exports.SignInButton = class SignInButton extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    let signin;
+    if (this.props.isAuthenticated) {
+      signin = <a className="settings-button" href="/settings" aria-label="Settings">
```

Comment:
@flodolo A `aria-label` attr should be localized just like any other attr?

Reply:
Yes, it should

Reply:
@chenba @flodolo Do we have existing settings aria-label localized?
https://github.com/mozilla-services/screenshots/blob/master/server/src/pages/shotindex/view.js#L209

Trying to find where do we need to add localized strings for aria-label?

Reply:
It certainly looks like the only existing `aria-label` is not localized.  Thanks for finding it.  Could you add that as well?